docs/using-connector.md

```diff
@@ -372,10 +372,27 @@ Any of the Spark Connector options can be provided in `.option()` or `.options()
 ### Key Based Partitioning
 Querying with the following methods with result in a RDD with single partition:
 
-* query2iRange(index, from, to)
 * query2iKeys(index, keys*)
 * queryBucketKeys(keys*)
 
+If 2i is Int, Long or BigInt, range query will be automatically split into a number of subranges defined by **spark.riak.input.split.count** option, creating a partition for each subrange.  
```

Comment:
If 2i is an Int, Long or BigInt, the range query will be automatically split into a number of subranges defined by the **spark.riak.input.split.count** option, creating a partition for each subrange.